api/src/play_asset_pack_manager.gd

```diff
@@ -69,7 +69,19 @@ func _ready():
 
 func _initialize():
 	_plugin_singleton = _initialize_plugin()
```

Comment:
Since Godot's Android Plugin also used this [singleton naming approach](https://docs.godotengine.org/en/stable/tutorials/plugins/android/android_plugin.html#loading-and-using-an-android-plugin), I think going with the current naming can be more consistent with Godot's doc.